templates/todo/projects/csharp-cosmos-sql/README.md

```diff
@@ -72,7 +72,8 @@ Here's a high level architecture diagram that illustrates these components. Noti
 
 !["Application architecture diagram"](assets/resources.png)
 
-> This template provisions resources to an Azure subscription that you will select upon provisioning them. Please refer to the [Pricing calculator for Microsoft Azure](https://azure.microsoft.com/pricing/calculator/) and, if needed, update the included Azure resource definitions found in `infra/main.bicep` to suit your needs.
+### Costing
```

Comment:
Maybe the header should just be "Cost of provisioning and deploying this template"